api/ttn/lorawan/v3/configuration_services.proto

```diff
@@ -35,6 +35,8 @@ message ListFrequencyPlansRequest {
   uint32 base_frequency = 1;
   // Optional Band ID to filter the results.
   string band_id = 2;
+  // Optional field to filter out the non-gateway related results.
+  bool gateways_only = 3;
```

Comment:
`Optional field to include only gateway related results`.

Reply:
Fixed.